### PR TITLE
spellcheck across the repository

### PR DESCRIPTION
**Pull request by xrstf**
## Summary
This is the result of running https://github.com/crate-ci/typos without any configuration on the kcp repository.

### CODE REVIEW

**Review by stevekuznetsov — APPROVED** [review]

**Comment by stevekuznetsov** [comment]
@ncdc will need your stamp ;)

File: contrib/crds/core/_persistentvolumes.yaml [hunk 1]
```diff
@@ -425,7 +425,7 @@ spec:
                 description: Local represents directly-attached storage with node affinity
                 properties:
                   fsType:
-                    description: Filesystem type to mount. It applies only when the Path is a block device. Must be a filesystem type supported by the host operating system. Ex. "ext4", "xfs", "ntfs". The default value is to auto-select a fileystem if unspecified.
+                    description: Filesystem type to mount. It applies only when the Path is a block device. Must be a filesystem type supported by the host operating system. Ex. "ext4", "xfs", "ntfs". The default value is to auto-select a filesystem if unspecified.
```
[the remark this answers is not in the document]
Comment (xrstf) [reply contrib/crds/core/_persistentvolumes.yaml:428]: Yeah I was unsure as well, I can revert it if you like :)

[the remark this answers is not in the document]
Comment (ncdc) [reply contrib/crds/core/_persistentvolumes.yaml:428]: Let's revert, thanks

[the remark this answers is not in the document]
Comment (xrstf) [reply contrib/crds/core/_persistentvolumes.yaml]: Done.

**Pull request merged**